pysmt/cmd/installers/base.py

```diff
@@ -37,6 +41,36 @@ def TemporaryPath(path):
         sys.path = old_path
 
 
+def delete_module(modname, paranoid=None):
```

Comment:
This function is a leftover from my previous experiments before subprocess, I guess it can be removed